automation-cypress-old/.gitignore

```diff
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+*.swp
+cypress/videos
+
```

Comment:
Do we need to keep `automation-cypress-ld/` around. Would prefer to just delete it if possible

Reply:
will move it to a seperate repo somewhere